payments-core/src/main/java/com/stripe/android/payments/core/authentication/DefaultPaymentAuthenticatorRegistry.kt

```diff
@@ -152,7 +135,7 @@ internal class DefaultPaymentAuthenticatorRegistry @Inject internal constructor(
             uiContext: CoroutineContext,
             threeDs1IntentReturnUrlMap: MutableMap<String, String>
         ): PaymentAuthenticatorRegistry {
-            val injectorKey = WeakSetInjectorRegistry.nextKey()
```

Comment:
Do the Components still need the `injectorKey`?

Reply:
Yes, it's actually a injected int that can be used anywhere when needed, e.g [here](https://github.com/stripe/stripe-android/blob/6bf3031b3934642737847a74f3c2af829832335b/payments-core/src/main/java/com/stripe/android/payments/core/authentication/threeds2/Stripe3DS2Authenticator.kt#L27) and it's used as parameter in the `DefaultFlowController` constructor in this PR